### ISSUE TITLE
.NET Core azure-pipelines.yml missing

### ISSUE THREAD

**Issue by bearmannl** [MicrosoftDocs/vsts-docs#4132]
The .NET Core project on gitHub is missing an azure-pipelines.yml file. As explained in the commit by Vijayma (https://github.com/MicrosoftDocs/pipelines-dotnet-core/commit/27d8fab9d8a851a4d37ff6c6a7d1fca720a1ab26) that file is no longer needed 'because docs explains how to create one'. That means this page is no longer correct for the .NET Core quickstart reference, one would still need to create their own azure-application.yml in order to get started, while it was meant as a quickstart to see a live working template before starting one on your own.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 02b747eb-53c2-dd5f-cd2b-f337c15e0864
* Version Independent ID: 04538505-00cd-33ac-8008-4b85cca089e3
* Content: [Create your first pipeline - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/create-first-pipeline?view=azure-devops&tabs=tfs-2018-2#feedback)
* Content Source: [docs/pipelines/create-first-pipeline.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/create-first-pipeline.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @andyjlewis
* Microsoft Alias: **alewis**

**Comment by vijayma** [comment]
When you follow the steps in the quickstart, the wizard automatically analyzes the repo and generate a azure-pipelines.yml file for you. So, you do not have write one manually.
